plugins/spu2-x/src/spu2sys.cpp

```diff
@@ -1080,15 +1083,16 @@ static void __fastcall RegWrite_VoiceAddr(u16 value)
 
         // NextA has been confirmed to not be allowed to be written to, so code has been commented out.
         // FlatOut & Soul Reaver 2 trigger these cases, but don't produce issues enabled or disabled.
+        // Enabled cases again due a regression in Wallace And Gromit: Curse Of The Were-Rabbit.
```

Comment:
Can you leave a note that mentions which cases are now uncommented (4 and 5 :p)?

(Also, pretty interesting that this works to fix the game..)

Reply:
I pretty much did in the last two commits
https://github.com/PCSX2/pcsx2/commit/eb955592ee0c473d55aa4164f6873d1ca7923a3a#diff-e5255967576cda2bc7f8be5c3efb5fdbR1083

Reply:
Maybe I can adjust the comment a bit tho.

Reply:
Should be better now.

Reply:
Thanks :)